src/main/scala/org/ergoplatform/nodeView/state/UtxoState.scala

```diff
@@ -195,12 +196,12 @@ class UtxoState(override val persistentProver: PersistentBatchAVLProver[Digest32
             new UtxoState(persistentProver, idToVersion(fb.id), store, constants)
           }
         }
-        stateTry.recoverWith[UtxoState] { case e =>
+        stateTry.recoverWith[UtxoState] { case NonFatal(ex) =>
```

Comment:
@kushti `Try` should not contain `Fatal` exceptions but this one actually is OOM error, I have to figure out how it gets there